packages/commerce-sdk-react/src/api-client/__mocks__/einstein.js

```diff
@@ -0,0 +1,62 @@
+/*
```

Comment:
Everything in the `api-client` folder is essentially copied from `template-retail-react-app/app/commerce-api/`